src/assets/compare.svg

```diff
@@ -0,0 +1,6 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I didn't see anything weird with Compare myself, but Favorite would occasionally be a jumble of black and white.

I did Object > Outline Stroke to the Compare and Favorite buttons in Figma, and that seems to have simplified their geometry. If you export them now, they should make much smaller SVGs and not render weirdly.